packages/core/core/package.json

```diff
@@ -21,7 +21,7 @@
     "@parcel/plugin": "^2.0.0",
     "@parcel/types": "^2.0.0",
     "@parcel/utils": "^2.0.0",
-    "@parcel/watcher": "^2.0.0",
+    "@parcel/watcher": "^2.0.0-alpha.3",
```

Comment:
I wonder if we should be versioning the other packages in core as something like this so we don't publish 2.0 before it's ready